internal/clientconn/conn_metrics.go

```diff
@@ -52,18 +47,18 @@ func NewMetrics() *Metrics {
 }
 
 // Describe implements prometheus.Collector.
-func (lm *Metrics) Describe(ch chan<- *prometheus.Desc) {
+func (lm *ConnMetrics) Describe(ch chan<- *prometheus.Desc) {
```

Comment:
It should be `cm` now, not `lm` (Listener Metrics)

Reply:
done